incapsula/resource_account.go

```diff
@@ -84,7 +84,7 @@ func resourceAccount() *schema.Resource {
 				Type:         schema.TypeString,
 				Default:      "true",
 				Optional:     true,
-				ValidateFunc: validation.StringInSlice([]string{"true", "false", "default"}, false),
```

Comment:
please add deprecation warning as described in documentation:
https://www.terraform.io/plugin/sdkv2/best-practices/deprecations#provider-attribute-removal:~:text=or%20Resource%20Rename-,%C2%BB,Provider%20Attribute%20Removal,-The%20recommended%20process